source/AMDSCentralServer.cpp

```diff
@@ -133,3 +108,36 @@ void AMDSCentralServer::onHundredMillisecondTimerUpdate(){
 
 	amptek1BufferGroup_->append(oneSpectralDataHolder);
 }
+
+void AMDSCentralServer::initializeBufferGroup(quint64 maxCountSize)
+{
+	QList<AMDSAxisInfo> mcpBufferGroupAxes;
+	mcpBufferGroupAxes << AMDSAxisInfo("X", 1024, "X Axis", "pixel");
+	mcpBufferGroupAxes << AMDSAxisInfo("Y", 512, "Y Axis", "pixel");
```

Comment:
@dretrex  this is only example buffer data. It doesn't have any meaningful usage in real environment. Technically, this section of codes will be reworked in the coming stories.